package.json

```diff
@@ -12,7 +12,7 @@
   "license": "MIT",
   "devDependencies": {
     "@types/supertest": "^2.0.5",
-    "ava": "^2.1.0",
+    "ava": "^3.5.0",
```

Comment:
Try `npm i -D ava@^3.5.0 nyc@^15.0.0 typescript@^3.8.3` combination of versions instead please? Upgrading ava alone breaks coverage for me:

```
$ npm test
> tsc && nyc ava && nyc report --reporter=html && tslint 'src/**/*.ts'



  4 tests passed

----------|----------|----------|----------|----------|-------------------|
File      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
----------|----------|----------|----------|----------|-------------------|
All files |        0 |        0 |        0 |        0 |                   |
----------|----------|----------|----------|----------|-------------------|
```